src/Raven.Client/Documents/Session/DocumentSession.Load.cs

```diff
@@ -52,6 +54,22 @@ public T Load<T>(string id)
             return loadOperation.GetDocuments<T>();
         }
 
+        public T Load<T>(string id, Action<IIncludeBuilder<T>> includes)
+        {
+            return Load(new []{id}, includes).Values.FirstOrDefault();
+        }
+
+        public Dictionary<string, T> Load<T>(IEnumerable<string> ids, Action<IIncludeBuilder<T>> includes)
+        {
+            var includeBuilder = new IncludeBuilder<T>();
```

Comment:
Let's not create IncludeBuilder when action is null

Reply:
fixed